src/pages/resultsView/download/CaseAlterationTable.tsx

```diff
@@ -93,13 +118,28 @@ export function generateOqlDisplayValue(oqlData: {[oqlLine: string]: IOqlData},
         const datum = oqlData[oqlLine];
 
         if (datum) {
-            oqlValue = datum.sequenced ? generateOqlValue(oqlData[oqlLine]) : "N/P";
+            oqlValue = datum.sequenced ? generateOqlValue(oqlData[oqlLine], alterationType) : "no alteration";
         }
     }
 
     return oqlValue;
 }
 
+export function findAlterationType(oql: OQLLineFilterOutput<AnnotatedExtendedAlteration>, alterationData: ICaseAlteration[]): string[]
```

Comment:
I dont like the word "find" here, it's not really finding anything is it? It's more like, computeAlterationTypes (since its finding more than one type).

also I don't see that the parameter oql is used?